<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenLoginWidget</class>
 <widget class="QWidget" name="ScreenLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="LoginFrame">
     <property name="minimumSize">
      <size>
       <width>524</width>
       <height>535</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>524</width>
       <height>535</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_19">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="menuFrame">
        <property name="minimumSize">
         <size>
          <width>524</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>524</width>
          <height>35</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_71">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_72">
           <item>
            <spacer name="horizontalSpacer_21">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="loginMiniButton">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Minimize</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loginCloseButtton">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Close</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QFrame" name="switchFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelError_1">
           <property name="text">
            <string>Your user name or password is wrong!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="commonUserFrame">
             <property name="minimumSize">
              <size>
               <width>376</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>376</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="userNameFrame">
                <property name="minimumSize">
                 <size>
                  <width>26</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="userNameLabel">
                <property name="minimumSize">
                 <size>
                  <width>54</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>54</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="text">
                 <string>User Name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="commonCombobox">
                <property name="minimumSize">
                 <size>
                  <width>256</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>256</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="topMargin">
            <number>10</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="commonKeyFrame">
             <property name="minimumSize">
              <size>
               <width>376</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>376</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="userKeyFrame">
                <property name="minimumSize">
                 <size>
                  <width>26</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="userKeyLabel">
                <property name="minimumSize">
                 <size>
                  <width>54</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>54</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="text">
                 <string>Password:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="userKeyLineEdit">
                <property name="minimumSize">
                 <size>
                  <width>256</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>256</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_63">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>65</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>74</number>
           </property>
           <item>
            <widget class="QCheckBox" name="rememberKeyCheckBox">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>98</width>
               <height>24</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Remember Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoLoginCheckBox">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>98</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Auto Login</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_20">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>88</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>15</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="commonUserLoginButton">
             <property name="minimumSize">
              <size>
               <width>376</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>376</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="langTypeCombo">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="langTypeLabel">
           <property name="text">
            <string>TXT_HELLO_WORLD</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>38</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_57">
        <property name="leftMargin">
         <number>80</number>
        </property>
        <property name="rightMargin">
         <number>79</number>
        </property>
        <item>
         <widget class="QLabel" name="loginTitleLabel">
          <property name="minimumSize">
           <size>
            <width>365</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>365</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>Distributed Visual Integrated Application System </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loginMiniButton</tabstop>
  <tabstop>loginCloseButtton</tabstop>
  <tabstop>autoLoginCheckBox</tabstop>
  <tabstop>commonUserLoginButton</tabstop>
  <tabstop>userKeyFrame</tabstop>
  <tabstop>userNameFrame</tabstop>
  <tabstop>userKeyLineEdit</tabstop>
  <tabstop>rememberKeyCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
